### PR TITLE
homework

### PR DESCRIPTION
**Pull request by vmaksimkina**

### CODE REVIEW

File: main_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestIsValidNumber(t *testing.T) {
```
Comment (zayac) [inline main_test.go:7]: Can you add a test for an empty string with a valid base, e.g. `{num: "", base: 2}`?

**Comment by zayac** [comment]
Where is `go.mod` file?
